build.gradle

```diff
@@ -13,6 +13,7 @@ buildscript {
 
 plugins {
     id 'com.gradle.plugin-publish' version '0.10.0' apply false
+    id 'com.dorongold.task-tree' version '1.5'
```

Comment:
What does this do and where do we need it?

Reply:
That was something I added to allow you to view the task tree in gradle. If you do ./gradlew ... taskTree you get a nice output of the task dependencies in gradle. Like this ....

:quasar-core:build
+--- :quasar-core:assemble
|    +--- :quasar-core:jar
|    |    \--- :quasar-core:classes
|    |         +--- :quasar-core:compileJava
|    |         |    \--- :quasar-core:processResources
|    |         \--- :quasar-core:processResources
|    +--- :quasar-core:javadocJar
|    |    \--- :quasar-core:javadoc
|    |         \--- :quasar-core:classes
|    |              +--- :quasar-core:compileJava
|    |              |    \--- :quasar-core:processResources
|    |              \--- :quasar-core:processResources
|    +--- :quasar-core:jdk7ShadowJar
|    |    +--- :quasar-core:classes
|    |    |    +--- :quasar-core:compileJava
.....